game_api.txt

```diff
@@ -381,7 +381,8 @@ The farming API allows you to easily register plants and hoes.
 		steps = 8,                             -- How many steps the plant has to grow, until it can be harvested
 		-- ^ Always provide a plant texture for each step, format: modname_plantname_i.png (i = stepnumber)
 		minlight = 13,                         -- Minimum light to grow
-		maxlight = default.LIGHT_MAX           -- Maximum light to grow
+		maxlight = default.LIGHT_MAX,          -- Maximum light to grow
+		soil_group = "soil"                    -- Soil node group where plant can grow ("soil" by default)
```

Comment:
Please also document that the group rating must be at least 2. (Otherwise I would expect any nonzero rating to work.)